app/components/InfoTable.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+import React from 'react';
+import InfoRow from './InfoRow';
+
+function getInfoRow(item) {
+  return <InfoRow {...item}/>;
+}
+
+function getInitialState() {
+   return {
+     expanded: false
+   };
+}
+
+const unexpandedListSize = 6;
+
+class InfoTable extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = getInitialState();
+    this.handleExpandClick = this.handleExpandClick.bind(this);
+  }
+
+  handleExpandClick() {
+    this.setState({
+      expanded: !this.state.expanded
+    });
+  }
+
+  getExpandButton() {
+    if ((this.props.items.length > unexpandedListSize) && this.props.expandable) {
+      let content;
+      if (this.state.expanded) {
+        content = ['View less ', <i className='fa fa-lg fa-chevron-up'/>];
+      }
+      else {
+        content = ['View more ', <i className='fa fa-lg fa-chevron-down'/>];
+      }
+      return (
+        <button className="button-expand" onClick={this.handleExpandClick}>
+          {content}
+        </button>
+      );
+    }
+    return [];
+  }
+
+  getFilteredItems(items) {
+    if (!this.props.expandable) {
+      return items;
+    }
+    if (unexpandedListSize > items.length) {
+      return items;
+    }
+    if (this.state.expanded) {
+      return items;
+    }
+    return items.slice(0, unexpandedListSize);
+  }
+
+  getItemsByColumn() {
+    const items = this.getFilteredItems(this.props.items);
+    let halfLength = Math.ceil(items.length / 2);
+    return [items.slice(0, halfLength), items.slice(halfLength)];
+  }
+
+  render() {
+    let itemsColumnOne, itemsColumnTwo;
+    [itemsColumnOne, itemsColumnTwo] = this.getItemsByColumn();
```

Comment:
Dividing items to two columns like this feels a bit bad. But what's even worse is that it screws the order on smaller screens. It would be better if the split to two columns could be done in CSS.